src/io/filesystem.ts

```diff
@@ -77,11 +85,20 @@ export async function getSystemPath(rsPath: string) {
  * @param absolutePath an operating-system absolute path
  * @returns
  */
-export async function getRefStudioPath(absolutePath: string) {
-  const baseDir = await getProjectBaseDir();
+export function getRefStudioPath(absolutePath: string) {
+  const baseDir = getProjectBaseDir();
   return absolutePath.replace(new RegExp(`^${baseDir}`), '');
 }
 
+/**
+ * Returns the the OS-specific separator
+ *
+ * @returns the separator (/ or \)
```

Comment:
```suggestion
 * @returns the separator (/ or \\)
```
nit: escape the backslash so it properly shows up in tooltip